draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -437,60 +438,66 @@ is later performed by an established connection.</t>
+            In some implementations, the decision to enter the
+            Unvalidated Phase could require coordination with the management
+            of buffers in the interface to the higher layers.</t>
+
         </list></t>
 
         <t>When a sender confirms the path and it
         receives an acknowledgement for the initial data without reported congestion,
         it MAY then enter the Unvalidated Phase.
         This transition occurs when a sender has more data than permitted
         by the current CWND.</t>
-
-        <t>Implementation requirements are provided in <xref target="req-"></xref>.</t>
-
+        
         <t>When a path is not confirmed, Careful Resume is not used
         and the sender enters the Normal Phase.</t>
 
+        <t>Implementation requirements are provided in <xref target="req-recon"></xref>.</t>
+
     </section> <!-- End of Reconnaissance Phase -->
      
     <section anchor="unv-phase" title="Unvalidated Phase">
         <t>The Unvalidated Phase is designed to enable the CWND
-        to more rapidly get up to speed, but this requires data to send.</t>
+        to more rapidly get up to speed, but this requires data to send.
+	</t>
+	    
         <t> The sender only enters this phase when there are saved CC parameters for the same
-        pair of endpoints and this information is currently valid (i.e., the saved parameters have
-        not expired).
+        pair of endpoints and this information is currently valid 
+	<xref target="req-lifetime"></xref>).
         A receiver can use a method (such as the QUIC BDP Frame
         <xref target="I-D.kuhn-quic-bdpframe-extension"></xref>)) to request
         that the sender does not enter this phase.</t>
 
-	    
-        <t>If the application is data limited, the sender sends insufficient data
-        to be able to validate transmission at the higher rate.
-        Careful Resume therefore remains in the Reconnaissance Phase and does not
-        transition to the Unvalidated Phase until the sender
-        has more data ready to send in the transmission buffer
-        than is permitted by the current CWND.
-        In some implementations, the decision to enter the
-        Unvalidated Phase could require coordination with the management
-        of buffers in the interface to the higher layers.</t>
-
-        <t>This phase paces transmission using an increased CWND (jump_CWND)
+        <t>This phase paces transmission using an increased CWND (jump_cwnd)
         that is calculated based on the saved CC parameters and current_RTT.</t>
         <t><list style="symbols">
              
              <t>Unvalidated Phase (jump_cwnd):
-             To avoid starving other (potential) flows that could have started
+             To avoid starving other flows that could have either started
              or increased their used capacity after the Observation Phase,
              the jump_cwnd MUST be no more than half of the saved_cwnd.
```

Comment:
This sentence should be updated if we think that during the unvalidated phase the jump_cwnd can be set to the saved_cwnd. this is what I undersood from what other modifications in this PR propose.

Reply:
That was not the intention, I have fixed the maths to match the sentence before.